CouchbaseLite.xcodeproj/xcshareddata/xcschemes/CBL ObjC.xcscheme

```diff
@@ -51,6 +50,23 @@
             ReferencedContainer = "container:CouchbaseLite.xcodeproj">
          </BuildableReference>
       </MacroExpansion>
+      <EnvironmentVariables>
+         <EnvironmentVariable
+            key = "CBL_TEST_HOST"
+            value = "localhost"
+            isEnabled = "NO">
+         </EnvironmentVariable>
+         <EnvironmentVariable
+            key = "CBL_TEST_PROXY_HOST"
+            value = "127.0.0.1"
+            isEnabled = "NO">
+         </EnvironmentVariable>
+         <EnvironmentVariable
+            key = "CBL_TEST_PROXY_PORT"
+            value = "8888"
+            isEnabled = "NO">
+         </EnvironmentVariable>
+      </EnvironmentVariables>
```

Comment:
@snej should these variable get checked-in?

Reply:
To make it easy to re-enable them, without having to look up the names in the source code (as I've had to do many times in the past week!) They're not enabled, so they do nothing unless you check the box.